css/style.css

```diff
@@ -1,4 +1,6 @@
 
+* {box-sizing: border-box;
```

Comment:
I would not advise to apply it to all tags with *. The reasons are that not tags have this property, not all tags need this change and it can badly influence performance.

Reply:
Ty for comment, next time i chose another way